README.md

```diff
@@ -1,2 +1,13 @@
 # timeseriesutils
-Utility functions for working with time series data in Python
+
+Utility functions for working with time series data in Python. Functionality is primarily oriented around converting time series data into a format suitable for prediction with methods for tabular data such as gradient boosting. This involves "featurizing" the data and creating columns with the prediction target(s).
+
+## Development environment
```

Comment:
It can a nice gesture to list any dependencies required to run the dev setup commands (especially in this case, since pdm may not be as familiar to people as conda).

Reply:
Thanks -- I just added some text about these [here](https://github.com/reichlab/timeseriesutils/pull/1/commits/a028609fd54e6bf51cfe4e5c027804e818817965).